src/meta/name.rs

```diff
@@ -256,7 +304,7 @@ mod test {
 
     #[test]
     fn test_order_impl_is_case_insensitive() {
-        let path_1 = Path::new("AAAA");
+        let path_1 = Path::new("/AAAA");
```

Comment:
Any reason why you changed this?

Reply:
There is no reason. I was concerned that this path is a relative path unlike other paths.